service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java

```diff
@@ -39,7 +39,7 @@ public RunImputationGcpJobFlight(FlightMap inputParameters, Object beanBag) {
         RunImputationJobFlightMapKeys.PIPELINE_INPUT_DEFINITIONS,
         RunImputationJobFlightMapKeys.PIPELINE_OUTPUT_DEFINITIONS,
         RunImputationJobFlightMapKeys.USER_PROVIDED_PIPELINE_INPUTS,
-        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_PROJECT,
+        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_BILLING_PROJECT,
         RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_NAME,
         RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_STORAGE_CONTAINER_URL,
```

Comment:
should we update this here or in a future PR when we start wiring through the inputs?

Reply:
oh good catch, should do it here.